text/utf8/module.f.ts

```diff
@@ -1,104 +1,246 @@
-import { flatMap, flat, stateScan, type List, type Thunk } from '../../types/list/module.f.ts'
-import type { StateScan } from '../../types/function/operator/module.f.ts'
-import type { Array1, Array2, Array3 } from '../../types/array/module.f.ts'
+import {
+  flat,
+  flatMap,
+  type List,
+  stateScan,
+  type Thunk,
+} from "../../types/list/module.f.ts";
```

Comment:
```suggestion
} from '../../types/list/module.f.ts'
```

We use `'` instead of `"` and no `;` at the end.